### PR TITLE
Added es6-babel-react-flux-karma

### PR DESCRIPTION
**Pull request by johnnyreilly**
To satisfy https://github.com/Microsoft/TypeScriptSamples/issues/48

### CODE REVIEW

File: es6-babel-react-flux-karma/gulp/staticFiles.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var gulp = require('gulp');
+var cache = require('gulp-cached');
+
+var targets = [
+  //{ description: 'FONTS', src: './fonts/*', dest: './dist/fonts' },
```
Comment (RyanCavanaugh) [inline es6-babel-react-flux-karma/gulp/staticFiles.js]: Remove?

Comment (johnnyreilly) [inline es6-babel-react-flux-karma/gulp/staticFiles.js]: Done

**Comment by johnnyreilly** [comment]
Done @RyanCavanaugh

**Comment by RyanCavanaugh** [comment]
Looks great!

In terms of sample structure, I think it'd be nice to have one or two additional "operations" (state changes) so that it's more clear what's boilerplate and what's part of the functionality. Maybe we can take that as a later PR.

**Comment by johnnyreilly** [comment]
Sounds good! And yes, happy to evolve the sample further in a subsequent PR. 

Let me know if you've anything particular in mind and I can put something together. Perhaps allow the creation of a list of people that you could greet? (That would be pretty much TodoMVC type functionality)

**Comment by mhegazy** [comment]
Thanks!

**Pull request merged**